lib/sign_up/signup_page.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:fpb/assets/fpb_icons/fpb_icons_icons.dart';
+import 'package:fpb/assets/fpb_svg.dart';
+import 'package:fpb/core/shared/presentation/theming/colors.dart';
+import 'package:fpb/sign_in/view/sign_in_page.dart';
+
+
+class SignUnPage extends StatefulWidget {
+  const SignUnPage({super.key});
+
+  @override
+  State<SignUnPage> createState() => _SignUnPageState();
+}
+
+class _SignUnPageState extends State<SignUnPage>
+    with SingleTickerProviderStateMixin {
+  late TabController tabController;
+  @override
+  void initState() {
+    super.initState();
+
+    tabController = TabController(
+      length: 2,
+      vsync: this,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    return Stack(
+      children: [
+        Positioned(
+          top: -size.height * .003,
+          child: SvgPicture.asset(
+            SvgNames.authBackground,
+            width: size.width,
+            height: size.height * 0.4,
+          ),
+        ),
+        Align(
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            height: size.height * .99,
+            padding: const EdgeInsets.all(20),
+            decoration: BoxDecoration(
+              color: Theme.of(context).backgroundColor,
+              borderRadius: const BorderRadius.only(
+                topLeft: Radius.circular(20),
+                topRight: Radius.circular(20),
+              ),
+            ),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.start,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                      'Join Us!',
+                      style: Theme.of(context).textTheme.displaySmall?.copyWith(
+                            color: AppColors.secondaryColorW,
+                            fontWeight: FontWeight.bold,
+                            fontSize: size.width * 0.043,
+                          ),),
```

Comment:
The same copywith was used in the sign-in screen. So you can make this modification once in the theme.